### DISCUSSION TITLE
Provide Error Pages, Divisions, or Popups for all HTTP errors received.

### DISCUSSION THREAD

**Discussion by da2ce7** [torrust/torrust-index-gui#143] [Feature requests]
Related issue: https://github.com/torrust/torrust-index-frontend/issues/99

**Comment by josecelano** [comment]
Hi @da2ce7, I think we now catch all errors. For example, when API is not available:

![image](https://github.com/user-attachments/assets/c301db3d-e8a7-478f-a1a3-7874f7d09f2d)

Anyway, we should probably overhaul the handling of errors. However, I would list concrete actions and maybe track those changes with an EPIC to have a collection of good practices to copy into other application parts.
